src/OpenVRTrackersDriver.cpp

```diff
@@ -96,7 +110,7 @@ bool openvr::DevicesManager::initialized() const
     return pImpl->vr && !std::string(pImpl->vr->GetRuntimeVersion()).empty();
 }
 
-bool openvr::DevicesManager::initialize()
+bool openvr::DevicesManager::initialize(std::string vrOrigin)
```

Comment:
Here you are passing by copy. You can use
```
const std::string& vrOrigin
```
to avoid creating a copy and simply passing a const reference.

Reply:
You need to add the ``const``, then it is possible to have default values :wink:

Reply:
Thanks for the tip! done in https://github.com/ami-iit/yarp-openvr-trackers/pull/12/commits/2c1a682b998a0d629495c96f3ded5661eff20e4a